components/ReusableSections/SponsorsSection/SponsorsSection.js

```diff
@@ -1,17 +1,50 @@
 import React from 'react';
-import Content from 'components/Content/Content';
+import Container from 'components/Container/Container';
+import Heading from 'components/Heading/Heading';
 import PartnerLogoLink from 'components/PartnerLogoLink/PartnerLogoLink';
-import partners from 'common/constants/partners';
+import partners, { PARTNER_TYPES } from 'common/constants/partners';
+import styles from './SponsorsSection.css';
+
+const isPaidSponsor = partner => partner.type === PARTNER_TYPES.PAID;
+
+const compareFunction = (first, second) => {
```

Comment:
Can you drop this and use `lodash/sortBy`?

Reply:
Also can you do it before exporting the constant, instead of twice in this render?

So like, in `common/constants/partners.js`

```javascript
import sortBy from 'lodash/sortBy';

const partners = [/* */];

export default sortBy(partners, ['name']);
```